### ISSUE TITLE
Bleno with Raspberry Pi 3 Model B+ not working very well but Model B works fine

### ISSUE THREAD

**Issue by Uniphix** [sandeepmistry/bleno#372]
Hello,

We have both Raspberry PI 3 Model B and B+.  When comparison between the two devices.  Our software will work just fine on the Model B+ which uses WIFI and Bluetooth.  When we install our software with Model B+ the Bluetooth will not work as the same as the Model B.

What I have found out is that on the Model B+ I can see the device show up on our application that is connecting/listening for.   When connected it will send a few bytes and also let the device know that it is still online (ping/pong).

However on our LOGS shows that it does not receive a App Ping from the mobile app (iOS).  When connected it will then try to get a list of wifi networks from the actual RPI which does show.  However it crashes our mobile application.  Bluetooth remains stuck, and the only way I can actually reset it completely is by rebooting the PI.

Basically commands like hciconfig hci0 reset or hciconfig hci0 down/up will result connection timeout issues.  Nothing I do will not cause it to work.

Is there something that I need to know about to make this work? With Model B+.  I do noticed that when we we're using Bluetooth Dongle at one time it would cause/do the same thing we're seeing again.  

I have no clue where to go from here. Any help will be much appreciated.

Thank you,
Ben

**Comment by Uniphix** [comment]
For anyone who may have the same issue.  Update your raspberry pi 3 B+ to the latest kernel, nodejs 8.11 (at least) and download/use bleno 0.5.0+ which somehow is what fixed my issue.
